File: qr-auth-frontend/src/app/_model/dto/customer-feedback-dto.ts
export class CustomerFeedbackDto {
  id!: number;
  name!: string;
  mobileNumber!: string;
  purchaseLocation!: string;
  opinion!: string;
  productUid!: string;

  constructor(obj?: Partial<CustomerFeedbackDto>) {
      Object.assign(this, obj);
  }

}

File: qr-auth-service/src/main/java/com/app/authentiScan/payload/dto/CustomerFeedbackDto.java
package com.app.authentiScan.payload.dto;

import lombok.*;

import javax.validation.constraints.NotBlank;
import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CustomerFeedbackDto {
    private Long id;
    private String name;
    @NotBlank
    private String mobileNumber;
    private String purchaseLocation;
    private String opinion;
    @NotBlank
    private String productUid;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
